src/PolicyEditor.js

```diff
@@ -256,6 +256,45 @@ export class PolicyEditor extends Component {
             }
         }
 
+        if (policy.actions) {
+            if (policy.actions.beforeSnapshotRoot) {
```

Comment:
can you add helper function to avoid repeated code?

Reply:
Done, I was just lazy there...